### DISCUSSION TITLE
extract state info and continue simulation

### DISCUSSION THREAD

**Discussion by ytimber** [InteractiveComputerGraphics/SPlisHSPlasH#160] [Q&A]
Hi,

I'm wondering whether SPlisHSPlasH will be able to do the following thing: extract information from the current state, such as velocity and position, and simulate directly from the extracted information one frame further, without involving any file I/O? Thank you!!

**Comment by janbender** [comment]
I didn't really get what you want to do. Can you give me some more details?
